storage/src/journal/authenticated.rs

```diff
@@ -419,7 +398,7 @@ where
             variable::Journal::init(context.with_label("journal"), journal_cfg).await?;
 
         // Rewind to last matching operation.
-        rewind(&mut journal, rewind_predicate).await?;
+        journal.rewind_to(rewind_predicate).await?;
```

Comment:
`rewind` used to `sync` the journal but `rewind_to` doesn't so I think we need to do that here